src/compiler/utilities.ts

```diff
@@ -12074,7 +12074,7 @@ export function forEachDynamicImportOrRequireCall<IncludeTypeSpaceImports extend
     cb: (node: CallExpression | (IncludeTypeSpaceImports extends false ? never : JSDocImportTag | ImportTypeNode), argument: RequireStringLiteralLikeArgument extends true ? StringLiteralLike : Expression) => void,
 ): void {
     const isJavaScriptFile = isInJSFile(file);
-    const r = /import|require/g;
+    const r = /import|defer|require/g;

```

Comment:
I'm not sure we need to expand this to include `defer` as that will hit more than a few false positives and slow down parsing existing code. As an alternative, we could modify `getNodeAtPosition` (which is only used by this function) to stop and return `current` when `isMetaProperty(child)` is `true`.

Reply:
I first tried to get the parent from the node returned by `getNodeAtPosition`, just to discover that it's not set. I didn't notice that `getNodeAtPosition` is only used here, I'll update it :+1:

Reply:
> I first tried to get the parent from the node returned by `getNodeAtPosition`, just to discover that it's not set.

program.ts calls this function before bind, so `.parent` is not set. In fact, that callee explicitly calls `setParentRecursively` in the callback as this is an expected case.